lib/seahorn/BvOpSem2Context.hh

```diff
@@ -261,6 +274,9 @@ public:
   /// \f setMemReadRegister
   Expr loadValueFromMem(Expr ptr, const llvm::Type &ty, uint32_t align);
 
+  /// \brief join two memories according to semantics of memory manager
```

Comment:
What is the semantics of this operation? I was expecting the join to be conditional on some path condition, but it is not in the interface

Reply:
Fixed. 
1. `joinMemories` will take in a join `cond`  e.g. at a `select` operation
2. The rest of the path conditions are carried in `mem1` and `mem2`.